Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -606,40 +606,37 @@ def test_set_color_sample(self):
 
     def test_paint_theme_sample(self):
         eq = self.assertEqual
-        d = self.page
-        del d.paint_theme_sample
-        hs_tag = d.highlight_sample.tag_cget
+        hi = self.page
+        del hi.paint_theme_sample  # Delete masking mock.
+        hs_tag = hi.highlight_sample.tag_cget
         gh = idleConf.GetHighlight
         fg = 'foreground'
         bg = 'background'
```

Comment:
I don't think `fg` or `bg` is used anymore.

Reply:
Ditto.  My initial changes still used them, but in the end, the literals are only needed in one place, console_new, and I in-lined them there.  Gone.